shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -41,8 +41,8 @@
       {{/each}}
     </ul>
     <ul class="navbar {{#unless showNavbar}}hide-desktop{{/unless}}">
-      <li class="navitem-open-grain {{#if isCurrentRoute 'selectGrain'}}current{{/if}}"><a href="{{pathFor route='selectGrain'}}"><span>Open...</span></a></li>
-      <li class="navitem-create-grain {{#if isCurrentRoute 'newGrain'}}current{{/if}}"><a href="{{pathFor route='newGrain'}}">New...</a></li>
+      <li class="navitem-create-grain {{#if isCurrentRoute 'newGrain'}}current{{/if}}"><a href="{{pathFor route='newGrain'}}">Apps...</a></li>
```

Comment:
I would rather hold off on this rename until the app details pages are ready. I know we debated this at length and not everyone agrees, but I think it could create additional confusion if we change it now. And anyway, it's nice for things to change together, so that when the sidebar changes it provides a hint that the UI within has changed too and users should check it out.